<launch>

   <rosparam command="load" file="$(find can_sort)/config/sort.yaml" />
   <node name="object_detection" pkg="can_sort" type="object_detection.py"/>
   
</launch>